src/Command/PolicyVerificationCommand.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace EdisonLabs\PolicyVerification\Command;
+
+use EdisonLabs\PolicyVerification\Check\AbstractPolicyCheckBase;
+use EdisonLabs\PolicyVerification\Report;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Implements command for Policy verification.
+ */
+class PolicyVerificationCommand extends Command
+{
+
+  /**
+   * @var array
+   */
+    protected $data = [];
+
+    /**
+     * @var \Symfony\Component\Console\Style\SymfonyStyle
+     */
+    protected $io;
+
+    /**
+     * @var \EdisonLabs\PolicyVerification\Report
+     */
+    protected $report;
+
+    /**
+     * Sets a policy check to report.
+     *
+     * @param \EdisonLabs\PolicyVerification\Check\AbstractPolicyCheckBase $policyCheck The policy check object.
+     */
+    public function setPolicyCheck(AbstractPolicyCheckBase $policyCheck)
+    {
+        if (!$this->report) {
+            return;
+        }
+
+        $this->report->setCheck($policyCheck);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+
+        // Gets data parameter.
+        $data = $input->getOption('data');
+        $data = $this->getDataArray($data);
+
+        $this->data = $data;
+        $this->report = new Report($this->data);
+
+        // Checks and includes a specific policy check class in the report.
+        $specificClass = $input->getOption('class');
+        if ($specificClass) {
+            if (!class_exists($specificClass)) {
+                throw new RuntimeException(sprintf('Class %s does not exist', $specificClass));
+            }
+
+            $specificClass = new $specificClass();
+            $this->setPolicyCheck($specificClass);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('edisonlabs:policy-verification')
+            ->setDescription('Edison Labs Policy verification')
+            ->setHelp('This command allows you to get results from policy checks')
+            ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The output format: table, json', 'table')
+            ->addOption('data', null, InputOption::VALUE_REQUIRED, 'Pass custom data to the policy checks, which can be a file or a string containing JSON format')
+            ->addOption('class', null, InputOption::VALUE_REQUIRED, 'Specify a policy check class to be included in the list of checks to be reported')
+        ;
+    }
+
+    /**
+     * Converts and returns the data JSON parameter value to array.
+     *
+     * @param string $data The data JSON value.
```

Comment:
This param can also be a path to a json file.
```suggestion
     * @param string $data The data command option.
```